product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx

```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
+     
+     - To allow the new superuser to connect using password authentication, add a line `pg_hba.conf` that allows `host` connections using `md5` authentication.
+
      - If you're using EDB Postgres Advanced Server, see [Modifying the pg_hba.conf file](/pem/latest/managing_database_server/#modifying-the-pg_hbaconf-file).
 
      - If you're using PostgreSQL, see [Client Authentication](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html).
 
-  3.  Verify that the sslutils extension and hstore contrib module are installed on your Postgres server.
+  4.  Verify that the sslutils extension and hstore contrib module are installed on your Postgres server.
 
       -   If you're using EDB Postgres Advanced Server, the sslutils extension and hstore contrib module are installed by default.
 
-      -   If you're using PostgreSQL, make sure you have access to the PostgreSQL community repository, and then install the sslutils extension and the hstore contrib module: 
+      -   If you're using PostgreSQL, make sure you have access to the PostgreSQL community repository ([yum](https://www.postgresql.org/download/linux/redhat/), [apt](https://wiki.postgresql.org/wiki/Apt)), and then install the sslutils extension and the hstore contrib module: 
 
           For RHEL/AlmaLinux/Rocky Linux 8:
           
           ```shell
-          dnf install sslutils-<x> postgresql<x>-contrib
+          dnf install sslutils_<x> postgresql<x>-contrib
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
           For RHEL/OL/CentOS 7:
           
           ```shell
-          yum install sslutils_<X> postgresql<x>-contrib
+          yum install sslutils_<x> postgresql<x>-contrib
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
           For Debian/Ubuntu:
           
           ```shell
-          apt install postgresql-<X>-sslutils postgresql<x>-contrib
+          apt install postgresql-<x>-sslutils
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
           For SLES:
           
           ```shell
-          zypper install sslutils_<X> postgresql<x>-contrib
+          zypper install sslutils_<x> postgresql<x>-contrib
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
-      !!! Note
-          Debian 10 and Ubuntu 20 increased the requirements to accept the certificates due to security reason.
+      !!! Note For Debian and Ubuntu users     
+          The postgres community repository for apt (used by Debian and Ubuntu) does not include `sslutils` and therefore this must be installed from the EDB repository along with [PEM](/installing_pem_server_using_edb_repository/index.mdx).
```

Comment:
This probably isn't needed anymore as the repos are both installed.

Reply:
I think if they have installed Postgres using Postgres community repos then this note is required.

Reply:
@sonotley, If you agree with this, we can take this PR further if not, let's discuss this over call.

Reply:
agreed